test/test_custom_ops.py

```diff
@@ -8,10 +8,12 @@
 import re
 import typing
 
+import torch._custom_ops as custom_ops
+
 import torch.testing._internal.custom_op_db
 from functorch import make_fx
 from torch import Tensor
-from torch._custom_op.impl import custom_op
+from torch._custom_op.impl import custom_op, CustomOp
```

Comment:
We don't need to deprecate it because it is not public API. I will make a cleanup pass sometime in the future because there are existing users who use the APIs. Pointing to the new API is a good suggestion, I will do that in this PR.